src/styles/card.css

```diff
@@ -0,0 +1,82 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: var(--main-accent-color);
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  align-self: center;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 1.5;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 1.4;
+  color: var(--secondary-gray-color);
+  margin-bottom: 16px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  margin-bottom: 24px;
+}
+
+.card__rate-text {
+  font-size: 10px;
+  line-height: 1.6;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__priсe-text {
+  font-size: 12px;
+  line-height: 1.5;
+  color: var(--secondary-gray-color);
+}
+
+.card__price-value {
+  font-size: 16px;
+  line-height: 1.13;
+  font-weight: 700;
+}
+
+.card__buy-button {
+  text-decoration: none;
+  background-color: var(--blue-accent);
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 14px;
+  line-height: 1.14;
+  padding: 12px;
+  text-align: center;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
```suggestion
```

Reply:
I don`t see changes after this comment, make sure you pushed your changes to github